src/utils.js

```diff
@@ -1,5 +1,6 @@
 import { oneLine } from "common-tags";
 import delve from "dlv";
+import { Linter } from "eslint";
```

Comment:
Not sure about this, thinking I should pass down `eslintPath`to where it's needed and use `requireModule` to get `Linter` instead. Node caches `require` calls.